tests/common_for_tests.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+from affine import Affine
+import copy
+import telluric as tl
+from telluric.constants import WEB_MERCATOR_CRS
+
+
+def make_test_raster(value=0, band_names=[], height=3, width=4, dtype=np.uint16,
+                     crs=WEB_MERCATOR_CRS, affine=None):
+    if affine is None:
+        affine = Affine.translation(10, 12) * Affine.scale(1, -1)
+    shape = [len(band_names), height, width]
+    array = np.full(shape, value, dtype=dtype)
+    mask = np.full(shape, False, dtype=np.bool)
+    image = np.ma.array(data=array, mask=mask)
+    raster = tl.GeoRaster2(image=image, affine=affine, crs=crs, band_names=band_names)
+    return raster
+
+
+multi_values_16b = {
+    'green': 5500,
+    'red': 7700,
+    'nir': 10000,
+    'blue': 11800
+}
+
+
+def sensor_bands_info():
+    return copy.deepcopy({
```

Comment:
Why do we need a copy here? Perhaps we can return the plain dictionary. Also, if it's meant to be a constant, I wounder if we could name it `SENSORS_BANDS_INFO = {...}`

Reply:
Also, is this something that could be useful for the user? The module `common_for_tests` won't be shipped in a normal installation.

Reply:
from my experience if I return a dict it  can be modified and it influences other tests that use it

Reply:
what do you recommend to do with `common_for_tests`?

Reply:
Not sure, for the moment let's leave it like that